use-case-guides/workday-request-time-off/README.md

```diff
@@ -209,149 +186,346 @@ curl --location 'https://{{domain}}.workday.com/ccx/api/absenceManagement/v1/{{i
+2. Request time off: Executes the `/api/absenceManagement/v1/{{instance}}/workers/{{worker_id}}/requestTimeOff` API to request time off
+
+**Caveats**
+
+1. If an employee selects a time off plan without adequante time balance, you will get an error message for attempting to request time off from the REST API. You can personalize the error message as part of exception handling logic inside your iPaaS.
+2. An employee could specify a time off plan that they are not eligible for
+1. Current Date: used in the "as of date" in the request PTO action. Could be static.
+2. Email Address: we assume that a user is only requesting time off for themselves, and use the `{{user.email_addr}}` value from user roster profile of the employee calling this plugin. This would need to be changed if an employee is requesting time off on behalf of another employee.
+
+### Build Plugin in Creator Studio
+1. Create a Path in Creator Studio. You can copy the plugin details below
+    * Plugin Name: `🏝️ Request Time Off in Workday` 
+    * Plugin Description: `Allow employees to submit time off requests in Workday.`
+    ![Name and Description](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/crest-1.png)
+2. The first question should ask which time off plan the employee wants to use for their time off request.
+    * You can use **API 2** above to discover the eligible time off plans for any given employee group.
+    ![Select Time Of](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/crest-2.png)
+3. Ask what the starting date for your employee's time off request.
+    * This will be configured as a Date Slot inside Creator Studio
+    ![Select Date](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/crest-3.png)
+4. Ask if the employee wants to specify an end date and additional comments for their time off request.
+    *  If employee selects no, they will only take time off for a single busines day (8 hours). This is configurable in your iPaaS.
+    ![Select Other Options](./Plugin%20Template%20Request%20Time-Off%20in%20Workday%20081c4d522bf64bbead3697288dd46047/crest-4.png)
+5. If your employee selects no, you can configure an API action to submit your time off request.
+    * If you use the iPaaS code below, configure your API action as follows:
+    * Connector: Use your iPaaS connector here
+    * API Endpoint Path: `/workers/{{user.email_addr}}/time-off`
+    * Method: `POST`
+    * Headers: `Content-Type` : `application/json`
+    * Query Params: Leave Blank
+    * Request Body: Use the below JSON to configure your API action. The time off type id can be found from running **API 2** for an employee's email.
+    ```json
+    {
+        "start_date": "{{start_date}}",
+        "time_off_type_id": "REDACTED"
```

Comment:
Not sure why this is redacted. It's not sesnitive data... right

Reply:
provided example time off type id